### ISSUE TITLE
Overfitting while the loss in low level.

### ISSUE THREAD

**Issue by H-Fangs** [zylo117/Yet-Another-EfficientDet-Pytorch#353]
Hi, zylo117,
Thanks for your great job on efficientdet, I use the ReduceLROnPlateau as the default  lr_scheduler with the init lr 3e-4, and the batch_size is 20 for image_size 512 on d4 model, however, when the classification loss down to 0.25, the model begin to overfitting, i have checked my tranform and dataloader with no problem.So Is their any better suggestion for this problem? here is my loss plot.
![image](https://user-images.githubusercontent.com/47218540/83321603-5513d500-a283-11ea-860b-4a7d62302481.png)
![image](https://user-images.githubusercontent.com/47218540/83321606-59d88900-a283-11ea-9b47-01739bc27cc9.png)
![image](https://user-images.githubusercontent.com/47218540/83321607-5cd37980-a283-11ea-97e1-2635ed1597d2.png)

**Comment by zylo117** [comment]
it'd be better not to change image size if not necessary.
and training large network like d4, you should prepare more data than training d0 to prevent overfitting.

**Comment by H-Fangs** [comment]
ok, thanks, I will try!
